runtime/calamari/build.rs

```diff
@@ -14,10 +14,19 @@
 // You should have received a copy of the GNU General Public License
 // along with Manta.  If not, see <http://www.gnu.org/licenses/>.
 
+#[cfg(all(feature = "std", feature = "metadata-hash"))]
 fn main() {
-    substrate_wasm_builder::WasmBuilder::new()
-        .with_current_project()
-        .export_heap_base()
-        .import_memory()
-        .build()
+    substrate_wasm_builder::WasmBuilder::init_with_defaults()
+        .enable_metadata_hash("CALAMARI", 12)
```

Comment:
Seems it should be token symbol In that polkadot doc

Reply:
yep, thanks!